src/nvim/testdir/test_spell.vim

```diff
@@ -303,7 +304,7 @@ func Test_zz_sal_and_addition()
   mkspell! Xtest Xtest
   set spl=Xtest.latin1.spl spell
   call assert_equal('kbltykk', soundfold('goobledygoook'))
-  call assert_equal('kprnfn', soundfold('k�op�r�n�ven'))
+  call assert_equal('kprnfn', soundfold('kóopërÿnôven'))
```

Comment:
Not part of the patch but I saved the file with `set encoding=utf-8`.

Reply:
Then the added note is not true anymore however - better to keep it like Vim then I guess.

Reply:
Redid it with latin1 encoding in Vim.